components/wb-fieldflow/fieldflow.js

```diff
@@ -570,7 +570,7 @@ var componentName = "wb-fieldflow",
 
 		// Create the label
 		if ( isReq && useReqLabel ) {
-			labelPrefix += " class='required" + labelInvisible + "'";
+			labelPrefix += " class='required'";
```

Comment:
Ok, can you check if the label is the same as the heading that immediately preceding the fieldflow, then hide it otherwise show it.
But use the technique of a `aria-hidden="true"` instead of the CSS class `wb-inv` to avoid the screen reader to read the same text twice.

Reply:
As previously discussed, new property is added "showLabel" in order to have a way to hide/show the labels only for selects with inline button. Thanks